container/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -ex
+
+cp /home/V2X-Hub/src/tmx/TmxCore/tmxcore.service /lib/systemd/system/
+cp /home/V2X-Hub/src/tmx/TmxCore/tmxcore.service /usr/sbin/
+chmod +x /home/V2X-Hub/container/service.sh /home/V2X-Hub/container/wait-for-it.sh
+mkdir -p /var/www/plugins/MAP /var/www/plugins/.ssl
+chown plugin:www-data /var/www/plugins/.ssl
+openssl req -x509 -newkey rsa:4096 -sha256 -nodes -keyout /var/www/plugins/.ssl/tmxcmd.key -out tmxcmd.crt -subj "/CN= <127.0.0.1> " -days 3650
+chown plugin:www-data /var/www/plugins/.ssl/*
```

Comment:
@codygarver Looks like line 10 is a duplicate of line 8.  Probably one is support to be a chgrp as that is what it was before.

Reply:
Also I don't think the : syntax is correct here.  Are you able to test these?

Reply:
@rob-baily that's the correct chown syntax, and it looks right in the built image. However, openssl was missing a flag (which is fixed now) and I went ahead and combined all of the chown while I was there.